src/mgr/PyState.cc

```diff
@@ -99,12 +106,49 @@ ceph_send_command(PyObject *self, PyObject *args)
   Py_DECREF(set_fn);
 
   auto c = new MonCommandCompletion(completion, tag);
-  global_handle->get_monc().start_mon_command(
-      {cmd_json},
-      {},
-      &c->outbl,
-      &c->outs,
-      c);
+  if (std::string(type) == "mon") {
+    global_handle->get_monc().start_mon_command(
+        {cmd_json},
+        {},
+        &c->outbl,
+        &c->outs,
+        c);
+  } else if (std::string(type) == "osd") {
+    std::string err;
+    uint64_t osd_id = strict_strtoll(name, 10, &err);
+    if (!err.empty()) {
+      // TODO: raise exception
+      return nullptr;
```

Comment:
returning nullptr will give the user a "SystemError: error return without exception set", which is a bit unfriendly, but basically what we want (raise an error).  We should go through all these places and do proper exception raising in a separate change